### PR TITLE
Remove pagination from challenge phase split list API

### PR DESCRIPTION
**Pull request by S2606**
Fix https://github.com/Cloud-CV/EvalAI/issues/1155: pagination has been removed from the API

### CODE REVIEW

**Comment by S2606** [comment]
@deshraj Please review the PR

File: apps/challenges/urls.py [hunk 1]
```diff
@@ -23,8 +23,6 @@
         views.get_challenge_by_pk, name='get_challenge_by_pk'),
     url(r'challenge$', views.get_challenges_based_on_teams,
         name='get_challenges_based_on_teams'),
-    url(r'(?P<challenge_pk>[0-9]+)/challenge_phase_split$', views.challenge_phase_split_list,
```
Comment (RishabhJain2018) [inline apps/challenges/urls.py]: @S2606 We don't have to remove the endpoint. We need to remove only the pagination from the API.
So, please revert these changes.

File: apps/challenges/views.py [hunk 2]
```diff
@@ -352,25 +352,6 @@ def challenge_phase_detail(request, challenge_pk, pk):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
-@throttle_classes([AnonRateThrottle])
```
Comment (RishabhJain2018) [inline apps/challenges/views.py]: The PR remove the complete view function which should not be the case.
In order to dive into `pagination`, please read [here](http://www.django-rest-framework.org/api-guide/pagination/).

**Comment by deshraj** [comment]
@S2606 please let us know if you are working on this otherwise we would ideally close this.

**Comment by S2606** [comment]
please close the issue

**Comment by RishabhJain2018** [comment]
@deshraj I think we should close this.

**Comment by deshraj** [comment]
Thanks for letting us know @S2606. Closing it.
